ui/src/shared/components/InputClickToEdit.tsx

```diff
@@ -1,34 +1,67 @@
-import React, {Component, PropTypes} from 'react'
+import React, {PureComponent} from 'react'
 
-class InputClickToEdit extends Component {
+interface Props {
+  wrapperClass: string
+  value?: string
+  onChange?: (value: string) => void
+  onBlur: (value: string) => void
```

Comment:
This `onChange` and `onBlur` probably take `Event` types.

Reply:
They do take strings types. I can change the name of those props if they are too similar to the props on the HTML input